osmtm/templates/project.edit.mako

```diff
@@ -36,6 +36,43 @@
   </div>
   </div>
 </div>
+
+<!-- Modal -->
+<div class="modal fade" id="invalidateAllModal" tabindex="-1" role="dialog" aria-labelledby="invalidateAll">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">${_('Invalidate All')}</h4>
+      </div>
+      <div class="modal-body">
+        <p>
+          ${_("This will mark all tasks currently marked as 'done' as invalid. Please use this only if you are sure of what you are doing.")}
+        </p>
+        <div class="text-center">
+          <div class="form-group">
+            <textarea id="project_invalidate_comment" name="invalidate_all_comment" class="form-control" placeholder="${_('Leave a comment')}" rows="2"></textarea>
```

Comment:
I made some changes here: https://github.com/osm-in/osm-tasking-manager2/commit/4720f71438acc4bcaaa55223f7caf15c3272c198

I am not convinced that this is still ideal. Would appreciate more specific ideas to how to make this look nicer and be clearer in what it does.